src/olympia/bandwagon/admin.py

```diff
@@ -9,7 +9,7 @@ class CollectionAdmin(admin.ModelAdmin):
     fields = ('name', 'slug', 'uuid', 'listed', 'type', 'application',
               'default_locale', 'author')
     raw_id_fields = ('author',)
-    readonly_fields = ('name', 'slug', 'uuid')
+    readonly_fields = ('name', 'uuid')
```

Comment:
Can we have name editing too?  (It works for `Addon` so I suppose the same trick works here).  Follow-up if too complicated.